crates/ty_project/src/metadata/options.rs

```diff
@@ -801,6 +799,59 @@ impl FromIterator<(RangedValue<String>, RangedValue<Level>)> for Rules {
     }
 }
 
+#[cfg(feature = "schemars")]
```

Comment:
Nit: I'd prefer to keep this in its own `mod` (and in the same position, it makes attributing changes with git easier)